lib/ansible/modules/system/puppet.py

```diff
@@ -43,9 +43,10 @@
     description:
       - Puppet environment to be used.
   logdest:
-    description:
-      - Where the puppet logs should go, if puppet apply is being used.
-    choices: [ stdout, syslog ]
+    description: |
+      Where the puppet logs should go, if puppet apply is being used. 'all' 
```

Comment:
You can wrap code snippets in `C()` and it will be stylized like in-line code in the generated HTML docs.